README.rst

```diff
@@ -90,14 +94,15 @@ and overrides the ``configure()`` method to define all of the application compon
 
   public class WebAnalyticsApp extends AbstractApplication {
     static final String APP_NAME = "WebAnalyticsApp";
+    static final String APP_DESCRIPTION = "Data Analysis with an OLAP Cube";
```

Comment:
+1

###
Random auto-corrects and typos are my special gift to you. When I forward they are from others. 

> On Apr 30, 2015, at 6:21 PM, John Jackson <notifications@github.com> wrote:
> 
> In README.rst:
> 
> > @@ -90,14 +94,15 @@ and overrides the ``configure()`` method to define all of the application compon
> >  
> >    public class WebAnalyticsApp extends AbstractApplication {
> >      static final String APP_NAME = "WebAnalyticsApp";
> > +    static final String APP_DESCRIPTION = "Data Analysis with an OLAP Cube";
> It's needed to get a description in the UI.
> 
> —
> Reply to this email directly or view it on GitHub.
>